src/components/app/Home.css

```diff
@@ -214,11 +221,20 @@
   }
 }
 
-@media (max-width: 767px) {
+@media (min-width: 500px) and (max-width: 767px) {
```

Comment:
nit: I don't think you need this `min-width` media query here ?

Reply:
Exactly!